### PR TITLE
Fix/combined indicators

### PR DESCRIPTION
**Pull request by shreeyash07**
- Addresses #63 

## Changes
- Add changes in combined indicators

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

File: app/components/ProgressBar/index.tsx [hunk 1]
```diff
@@ -66,9 +74,15 @@ function ProgressBar(props: Props) {
...
-                    M
+                    {suffix}
                 </div>
             </div>
+            {avgResult.regionalPercentage
+                && (
+                    <div className={styles.regionalValue}>
+                        {`Regional ${avgResult.regionalPercentage}${suffix}`}
+                    </div>
+                )}
```
Comment (subinasr) [inline app/components/ProgressBar/index.tsx]:
```suggestion
            {avgResult.regionalPercentage && (
                    <div className={styles.regionalValue}>
                        {`Regional ${avgResult.regionalPercentage}${suffix}`}
                    </div>
                )}
```

File: app/views/Dashboard/Overview/MapView/index.tsx [hunk 2]
```diff
@@ -116,9 +117,6 @@ function MapView(props: MapViewProps) {
                         filtered={false}
                         errored={false}
                         pending={false}
-                        borderBetweenItem
-                        borderBetweenItemWidth="medium"
-                        borderBetweenItemClassName={styles.progressItemBorder}
```
Comment (subinasr) [inline app/views/Dashboard/Overview/MapView/index.tsx]: Why??
Let's keep this.

**Review by subinasr — CHANGES_REQUESTED** [review]

**Comment by subinasr** [comment]
Also, let's change the way we've handled the barName in the progress bars. Let's not change the dummy data to meet our needs because that may not be possible with actual data. Instead, let's figure out a way to make the existing data work by modifying our progress bar component.

File: app/views/Dashboard/Overview/MapView/index.tsx (hunk 2 above)
Comment (shreeyash07) [reply app/views/Dashboard/Overview/MapView/index.tsx]: done

File: app/components/ProgressBar/index.tsx (hunk 1 above)
Comment (shreeyash07) [reply app/components/ProgressBar/index.tsx]: Done

**Review by subinasr — APPROVED** [review]

File: app/components/ProgressBar/index.tsx [hunk 3]
```diff
@@ -1,17 +1,20 @@
...
-        countryName: string | undefined,
+        barName: string | undefined,
         id: string,
         title: string | undefined,
         color?: string | undefined,
         value: number | undefined,
+        regional?: number | undefined,
         totalValue: number | undefined,
     };
 }
```
Comment (AdityaKhatri) [inline app/components/ProgressBar/index.tsx]: flatten this out

File: app/components/ProgressBar/index.tsx [hunk 4]
```diff
@@ -21,13 +24,17 @@ function ProgressBar(props: Props) {
...
         () => ({
             percentage: progressData?.value && progressData?.totalValue
                 && (((progressData.value / progressData.totalValue) * 100).toFixed(1)
                     ?? undefined),
+            regionalPercentage: progressData.regional && progressData?.totalValue
+                && (((progressData.regional / progressData.totalValue) * 100).toFixed(1)
+                    ?? undefined),
         }),
         [progressData],
     );
```
Comment (AdityaKhatri) [inline app/components/ProgressBar/index.tsx]: separate this into 2 varibales

File: app/components/ProgressBar/index.tsx [hunk 5]
```diff
@@ -1,17 +1,20 @@
 import React, { useMemo } from 'react';
 import { _cs } from '@togglecorp/fujs';
+import { IoInformationCircleOutline } from 'react-icons/io5';
 
 import styles from './styles.css';
 
 export interface Props {
     className?: string | undefined;
     barHeight: number;
```
Comment (AdityaKhatri) [inline app/components/ProgressBar/index.tsx:9]: Either change this to different types or remove.

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

**Review by priyesh777 — APPROVED** [review]

File: app/components/ProgressBar/index.tsx (hunk 4 above)
Comment (shreeyash07) [reply app/components/ProgressBar/index.tsx]: Done

File: app/components/ProgressBar/index.tsx (hunk 5 above)
Comment (shreeyash07) [reply app/components/ProgressBar/index.tsx:9]: Requested changes done

File: app/views/Dashboard/CombinedIndicators/index.tsx [hunk 6]
```diff
@@ -4,20 +4,25 @@ import {
...
 
 import styles from './styles.css';
 
 const progressBarKeySelector = (d: ProgressBarRendererProps) => d.id;
 
-const barHeight = 15;
+const barHeight = 8;
 export interface ProgressBarRendererProps {
+    className?: string,
+    barHeight?: number,
```
Comment (AdityaKhatri) [inline app/views/Dashboard/CombinedIndicators/index.tsx:17]: remove this from props if not used

Comment (shreeyash07) [reply app/views/Dashboard/CombinedIndicators/index.tsx:17]: barHeight is being used in two different component with different height sizes.

**Review by subinasr — APPROVED** [review]

**Pull request merged**